services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -228,6 +249,10 @@ async def create_computation(
 
     except ProjectNotFoundError as e:
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=f"{e}") from e
+    except ClusterNotFoundError as e:
+        raise HTTPException(
```

Comment:
So ClusterNotFoundERror has a different detail message here as in a few lines above

THOUGHT: shouldn't we instead use the ``app.add_exception_handler`` mechanism to transform domain excpetions (e.g. ``Cluster*Error``) into HTTP exceptions? Or this is too specific to this handler? I am aware that the former apply at an application level but then we could create some decorator-based exception_handler ...